Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Spawnable/AssetPlatformComponentRemover.cpp

```diff
@@ -49,9 +49,14 @@ namespace AzToolsFramework::Prefab::PrefabConversionUtils
         prefabProcessorContext.ListPrefabs(
             [&prefabProcessorContext, &excludedComponents](PrefabDocument& prefab) -> void
             {
+                (void) prefabProcessorContext;
+                (void) excludedComponents;
```

Comment:
if these variables are not referenced in the call below why bring them into the lambda to begin with. passing it to void just silences the warning?

Reply:
Interestingly, the error I got for those two variables is gone, so I removed the lines. The other one is still necessary and as the title suggests only for release builds, because otherwise the variable is used.